src/deploy/increment-cachebuster.js

```diff
@@ -25,6 +25,5 @@ module.exports = (env) => {
             const nextIncrement = new Date();
             nextIncrement.setMinutes(nextIncrement.getMinutes() + 10);
             console.log(`cachebuster is set to increment at: ${nextIncrement.toLocaleString()}`);
-            resolve(body.current_cachebuster_value);
```

Comment:
this is in a promise chain, don't we still need the promise result? i.e.
```
return nextIncrement;

Reply:
cant resolve from within then. i can just return it for now, but the resolve is from before you refactored, and i had everything nested from within a promise.